packages/@aws-cdk/aws-servicecatalogappregistry/lib/application.ts

```diff
@@ -270,6 +285,10 @@ export class Application extends ApplicationBase {
     this.applicationId = application.attrId;
     this.applicationName = props.applicationName;
     this.nodeAddress = cdk.Names.nodeUniqueId(application.node);
+
+    this.applicationManagerUrl = new cdk.CfnOutput(this, 'ApplicationManagerUrl', {
+      value: `https://${this.env.region}.console.aws.amazon.com/systems-manager/appmanager/application/AWS_AppRegistry_Application-${this.applicationName}`,
```

Comment:
Add a `description` as well. Include the construct path in it.

Reply:
Addressed in the new revision.